src/System.Runtime.Extensions/tests/System/IO/Path.Combine.cs

```diff
@@ -2,288 +2,253 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.IO;
+using System.Collections.Generic;
+
 using Xunit;
 
-public class PathCombineTests
+namespace System.IO.Tests
 {
-    private static readonly char s_separator = Path.DirectorySeparatorChar;
-
-    [Fact]
-    public static void PathIsNull()
-    {
-        VerifyException<ArgumentNullException>(null);
-    }
-
-    [Fact]
-    public static void VerifyEmptyPath()
-    {
-        //paths is empty
-        Verify(new String[0]);
-    }
-
-    [Fact]
-    public static void VerifyPath1Element()
-    {
-        //paths has 1 element
-        Verify(new String[] { "abc" });
-    }
-
-    [Fact]
-    public static void VerifyPath2Elements()
-    {
-        //paths has 2 elements
-        Verify(new String[] { "abc", "def" });
-    }
-
-    [Fact]
-    public static void VerifyPathManyElements()
-    {
-        //paths has many elements
-        Verify(new String[] { "abc" + s_separator + "def", "def", "ghi", "jkl", "mno" });
```

Comment:
What happened to this test case, i.e. an element that contains a separator and others that don't?